src/current/_includes/cockroachcloud/quickstart/create-a-free-cluster.md

```diff
@@ -3,7 +3,11 @@
 1. If you haven't already, <a href="https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}" rel="noopener" target="_blank">sign up for a CockroachDB {{ site.data.products.cloud }} account</a>.
 1. [Log in](https://cockroachlabs.cloud/) to your CockroachDB {{ site.data.products.cloud }} account.
 1. On the **Clusters** page, click **Create Cluster**.
-1. On the **Create your cluster** page, select **Serverless**.
-1. Click **Create cluster**.
+1. On the **Select a tier** page, select **Serverless**.
```

Comment:
Looks like we went with "Select a plan" instead of "tier": https://www.figma.com/file/idjQ4EtJs78TuKTswKCTiW/Cloud-2.0?type=design&node-id=3513-98852&mode=design&t=0gNDZEm9SF3cYn8f-4

I'm fixing that in staging today :)